docs/libertyExtensions.md

```diff
@@ -1,16 +1,13 @@
 ## Liberty extension properties
 The Liberty Gradle plugin defines properties in the `liberty` block to customize task behavior.
-These properties are divided in three groups:
+These properties are divided in two groups:
 * The [general runtime properties](#general-runtime-properties) which control how the runtime is installed with the `installLiberty` task.
 * The [Liberty server configuration](#liberty-server-configuration) properties control how a server is initialized and the applications that are installed. These properties are defined in a `server` block within the `liberty` block.
-* The [dev extension properties](#dev-extension-properties) that are specific to dev mode when using containers. These properties are defined in a `dev` block within the `liberty` block.
 
-The [installLiberty](installLiberty.md) and [compileJSP](compileJsp.md) tasks can be configured with their related extension inside the `liberty` block.
+The [installLiberty](installLiberty.md) and [compileJSP](compileJsp.md) tasks can be configured with their related extension inside the `liberty` block.  The [libertyDevc](libertyDev.md#libertydevc-task-container-mode) task can be configured with its related extension inside the `dev` block.
```

Comment:
Combine these sentences into one.
The [installLiberty](installLiberty.md), [compileJSP](compileJsp.md) and [libertyDevc](libertyDev.md#libertydevc-task-container-mode) tasks can be configured with their related extension inside the `liberty` block.

Reply:
Fixed